apps/api/src/app/portfolio/portfolio.service.ts

```diff
@@ -1652,7 +1654,19 @@ export class PortfolioService {
       case 'ytd':
         portfolioStart = max([
           portfolioStart,
-          setDayOfYear(new Date().setHours(0, 0, 0, 0), 1)
+          startOfYear(new Date().setHours(0, 0, 0, 0))
+        ]);
+        break;
+      case 'mtd':
+        portfolioStart = max([
+          portfolioStart,
+          startOfMonth(new Date().setHours(0, 0, 0, 0))
+        ]);
+        break;
+      case 'wtd':
+        portfolioStart = max([
+          portfolioStart,
+          startOfWeek(new Date().setHours(0, 0, 0, 0))
```

Comment:
What do you think of `startOfWeek(new Date().setHours(0, 0, 0, 0), { weekStartsOn: 1 })`. At least as a European I would expect to get midnight between Sunday and Monday (not Saturday and Sunday).

Reply:
Yeah that makes sense, will update.